### ISSUE TITLE
Why the depth always miss some lower depth or high depth part after training of the Depth_Head

### ISSUE THREAD

**Issue by liyuan-qiu** [DepthAnything/Depth-Anything-V2#203]
![image](https://github.com/user-attachments/assets/db16aade-79a6-4ddd-aa7c-8e0422f4c326)
Hi, I am training the depth_head layer  of depth-anything v2 with my own dataset, but it seems like that the original vits parameter performance is good, but after several training, the depth always seems like miss some part.
 And I draw the line of the middle depth picture from the "ground truth", "vits" from depth-anything v2, "trained result" from my own training parameter as shown in the following picture.
![image](https://github.com/user-attachments/assets/24b7be61-11ba-432c-877f-4ca346a76a59)
I have tried different learning rate, different max/min_depth, but there always seems to be this kind of missing depth part. Can you give some hint about the missing depth?
